conf/ebi_codon.config

```diff
@@ -0,0 +1,60 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Nextflow config file for EMBL-EBI Codon Cluster
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Author: Saul Pierotti
+Mail: saul@ebi.ac.uk
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+params {
+    config_profile_contact = "Saul Pierotti (@saulpierotti-ebi)"
+    config_profile_description = "The European Bioinformatics Institute HPC cluster (codon) profile"
+    config_profile_url = "https://www.ebi.ac.uk/"
+}
+
+singularity {
+    enabled = true
+    // the default is 20 minutes and fails with large images
+    pullTimeout = "3 hours"
+}
+
+conda {
+    enabled = true
+    // use the faster mamba solver instead of conda
+    useMamba = true
+}
```

Comment:
enabling both?

Reply:
I use it like this usually so that I can run some processes with conda and some with singularity when developping my own pipelines. When running nf-core pipeline I usually use singularity enabling the appropriate profile. Do you think that it would it be better to keep just singularity in the config?